### PR TITLE
Remove SBT dependency from reloader

### PR DESCRIPTION
**Pull request by benmccann**
This is the first step towards supporting Maven + Play along with https://github.com/playframework/playframework/pull/7093

### CODE REVIEW

File: framework/src/run-support/src/main/scala/play/runsupport/AssetsClassLoader.scala [hunk 1]
```diff
@@ -5,7 +5,6 @@ package play.runsupport
 
 import sbt._
```
Comment (benmccann) [inline framework/src/run-support/src/main/scala/play/runsupport/AssetsClassLoader.scala]: The tests fail if I remove this, but I have no idea why. The code compiles without it. How do I figure out what magic this is doing?

**Comment by mkurz** [comment]
Ben you don't need to close/re-open pull requests to trigger a build.
You can just click "Restart build" directly in travis itself - even better you can also just use "Restart job" to restart failed jobs instead of the whole build (saves time).

**Comment by benmccann** [comment]
Ah thanks!! Super happy to learn that!

**Comment by mkurz** [comment]
Also check the "More options" dropdown on the right side - there you can e.g. clean the cache of your build. Sometimes this can be useful as well.

**Pull request merged**